src/translations/es.json

```diff
@@ -0,0 +1,21 @@
+{
+  "error-occured": "Un error ha impedido que la extensión funcione apropiadamente: ",
+  "version-update-message": "¡Ahora estás usando ezpp!",
```

Comment:
just to make sure, this is used in the context of version changes as in `You are now using ezpp! v.1.5.0`, which shows up when the extension is updated

Reply:
i should probs make it a template string to make it more clear

Reply:
Alright, I'll change it a bit so it looks better, the ¡ ! definitively look weird if the version will be after them